WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -284,6 +285,11 @@ class PostCoordinator: NSObject {
         topViewController.present(navigation, animated: true)
     }
 
+    func didResolveConflict(for post: AbstractPost) {
+        workers[post.objectID]?.error = nil
```

Comment:
Is this code still needed?

Reply:
I revised this to call `startSync(for:)` because we still need to clear the error - is that what you mean? https://github.com/wordpress-mobile/WordPress-iOS/pull/22992/commits/30bbdd72a9faa1c8910e44a40a8e1f4d490b3b12